sdk/cosmos/azure-cosmos-test/src/main/java/com/azure/cosmos/test/faultinjection/FaultInjectionServerErrorResult.java

```diff
@@ -13,10 +13,18 @@ public final class FaultInjectionServerErrorResult implements IFaultInjectionRes
     private final Integer times;
     private final Duration delay;
 
-    FaultInjectionServerErrorResult(FaultInjectionServerErrorType serverErrorTypes, Integer times, Duration delay) {
+    private final Boolean suppressServiceRequests;
```

Comment:
can be boolean here I guess

Reply:
No - null means default (depending on whether delay is larger than network timeout)

Reply:
yea I mean why not translate null -> default value, then we do not need to worry whether it is null or not internally

Reply:
We can't map to the default value because we don't have access to the network request timeout yet? And the default would vary on delay > network request timeout